### ISSUE TITLE
Custom cross-validation

### ISSUE THREAD

**Issue by atkm** [atkm/avazu-ctr#1]
The test set should contain the final day, and the training set should contain all days before the final.
The same goes for splitting the training set into train and dev.

**Comment by atkm** [comment]
Implemented in Model 1 - sklearn.ipynb. Might need to generalize it to accommodate later models.
